simde/arm/neon/cmla_lane.h

```diff
@@ -277,35 +262,33 @@ simde_float32x4_t simde_vcmlaq_lane_f32(simde_float32x4_t r, simde_float32x4_t a
 {
   #if defined(SIMDE_RISCV_V_NATIVE)
       simde_float32x4_private r_ = simde_float32x4_to_private(r),
-                              a_ = simde_float32x4_to_private(a);
-      simde_float32x2_private b_ = simde_float32x2_to_private(b);
+                              a_ = simde_float32x4_to_private(a),
+                              b_ = simde_float32x4_to_private(simde_vreinterpretq_f32_u64(simde_vdupq_n_u64(simde_uint64x1_to_private(simde_vreinterpret_u64_f32(b)).values[lane])));
       uint32_t idx1[4] = {0, 0, 2, 2};
       vfloat32m2_t a_tmp = __riscv_vlmul_ext_v_f32m1_f32m2 (a_.sv128);
       vfloat32m1_t op1 = __riscv_vlmul_trunc_v_f32m2_f32m1(__riscv_vrgather_vv_f32m2( \
         __riscv_vslideup_vx_f32m2(a_tmp, a_tmp, 4, 8), __riscv_vle32_v_u32m2(idx1, 4), 4));
-      r_.sv128 = __riscv_vfmacc_vf_f32m1(r_.sv128, b_.values[lane], op1, 4);
+      r_.sv128 = __riscv_vfmacc_vv_f32m1(r_.sv128, b_.sv128, op1, 4);
       return simde_float32x4_from_private(r_);
   #else
-    simde_float32x4_private r_ = simde_float32x4_to_private(r), a_ = simde_float32x4_to_private(a),
-                            b_ = simde_float32x4_to_private(simde_vdupq_n_f32(simde_float32x2_to_private(b).values[lane]));
+    simde_float32x4_t b_tmp_ = simde_vreinterpretq_f32_u64(simde_vdupq_n_u64(simde_uint64x1_to_private(simde_vreinterpret_u64_f32(b)).values[lane]));
     #if defined(SIMDE_SHUFFLE_VECTOR_)
+      simde_float32x4_private r_ = simde_float32x4_to_private(r), a_ = simde_float32x4_to_private(a),
+                              b_ = simde_float32x4_to_private(b_tmp_);
       a_.values = SIMDE_SHUFFLE_VECTOR_(32, 16, a_.values, a_.values, 0, 0, 2, 2);
       r_.values += b_.values * a_.values;
+      return simde_float32x4_from_private(r_);
     #else
-      SIMDE_VECTORIZE
-      for (size_t i = 0; i < (sizeof(r_.values) / (2 * sizeof(r_.values[0]))); i++)
-      {
-        r_.values[2 * i] += b_.values[lane] * a_.values[2 * i];
-        r_.values[2 * i + 1] += b_.values[lane] * a_.values[2 * i];
-      }
+      return simde_vcmla_f32(r, a, b_tmp_);
```

Comment:
MSVC rightfully complains here, r is of type "simde_float32x**4**_t" but `simde_vcmla_f32` needs a `simde_float32x2_t`

https://ci.appveyor.com/project/nemequ/simde/builds/51411171/job/j0i25ixsjy08vx6r#L1438